tools/generate-baseline.js

```diff
@@ -33,17 +33,31 @@ const baselineIds = new Map([
 	[false, BASELINE_FALSE],
 ]);
 
+/**
+ * Encodes the baseline status and year fields into a single string.
+ * @param {{status: string, year: number}} baselineStatus The baseline fields to encode.
+ * @returns {string} The encoded baseline status and year.
+ */
+function encodeBaselineStatus({ status, year }) {
```

Comment:
Can we change `status` and `year` to regular (not destructured) parameters? There's no need to create an object just to call this function.